### ISSUE TITLE
[vscjava.vscode-java-pack] Error: Cannot resolve bundled extension

### ISSUE THREAD

**Issue by jankeromnes** [open-vsx/publish-extensions#11]
```
 DONE  Packaged: /tmp/tmp-3689wm1l2gxLjTiC.vsix (16 files, 549.58KB)

[FAIL] Could not process extension: {
  "id": "vscjava.vscode-java-pack",
  "version": "0.8.1",
  "repository": "https://github.com/microsoft/vscode-java-pack"
}
Error: Cannot resolve bundled extension: VisualStudioExptTeam.vscodeintellicode
    at Object.<anonymous> (/home/runner/work/publish-extensions/publish-extensions/node_modules/ovsx/lib/publish.js:52:19)
    at Generator.next (<anonymous>)
    at fulfilled (/home/runner/work/publish-extensions/publish-extensions/node_modules/ovsx/lib/publish.js:14:58)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**Comment by jankeromnes** [comment]
Same error as #8.

**Comment by jankeromnes** [comment]
Merged into #8.
